Tasks/AzureRmWebAppDeployment/VariableSubstituter.ps1

```diff
@@ -0,0 +1,85 @@
+﻿function Substitute-Variables
+{  
+    Param(
+        [string][Parameter(mandatory=$true)] 
+        $WebAppFolderPath,
+        [string][Parameter(mandatory=$true)] 
+        $ConfigFileRegex
+    )
+
+    # Load modules
+    . $PSScriptRoot/XMLSubstituter.ps1 -Force
+
+    # Defined supported MIME type
+    $ApplicationXmlMimeType = "application/xml"
+    $TextXmlMimeType = "application/xml"
+    $ConfigExtension = ".config"
+
+    # Find all configuration files
+    $webConfigFiles = Get-ChildItem -Path $WebAppFolderPath -Filter $ConfigFileRegex -Recurse -ErrorAction SilentlyContinue -Force
+
+    if( $webConfigFiles.Count -eq 0)
+    {
+        Write-Verbose "File not found at location : $WebAppFolderPath with pattern $ConfigFileRegex"
```

Comment:
Should this be a warning? As user said variable substitution but in actual that didn't happen